api-schema/src/main/java/org/stellar/anchor/api/sep/SepAuthorizationExceptionResponse.java

```diff
@@ -0,0 +1,12 @@
+package org.stellar.anchor.api.sep;
+
+import lombok.Data;
+
+@Data
+public class SepAuthorizationExceptionResponse {
+  String error;
+
+  public SepAuthorizationExceptionResponse(String error) {
+    this.error = error;
+  }
+}
```

Comment:
I don't see any reason for adding this class, since it's the same as `SepExceptionResponse`:
https://github.com/stellar/java-stellar-anchor-sdk/blob/8cd8d07a4ff8ce202158c4c064cde542bbc4acc2/api-schema/src/main/java/org/stellar/anchor/api/sep/SepExceptionResponse.java#L5-L12